controller/area_blackbox_test.go

```diff
@@ -60,190 +59,204 @@ func (rest *TestAreaREST) UnSecuredController() (*goa.Service, *AreaController)
+			/*
+				TestAreaREST ---> TestSuccessCreateMultiChildArea-0 ----> TestSuccessCreateMultiChildArea-0-0
+			*/
+			// given
+			fxt := tf.NewTestFixture(t, rest.DB, tf.Areas(1))
+			parentArea := fxt.Areas[0]
+			parentID := parentArea.ID
+			ca := newCreateChildAreaPayload("TestSuccessCreateMultiChildArea-0")
+			owner := fxt.Identities[0]
+			svc, ctrl := rest.SecuredControllerWithIdentity(owner)
+			// when
+			resp, created := test.CreateChildAreaCreated(t, svc.Context, svc, ctrl, parentID.String(), ca)
+			// then
+			assert.Equal(t, *ca.Data.Attributes.Name, *created.Data.Attributes.Name)
+			assert.Equal(t, parentID.String(), *created.Data.Relationships.Parent.Data.ID)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.child1_ok.res.golden.json"), created)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.res.headers.golden.json"), resp.Header())
+			// Create a child of the child created above.
+			ca = newCreateChildAreaPayload("TestSuccessCreateMultiChildArea-0-0")
+			newParentID := *created.Data.Relationships.Parent.Data.ID
+			// when
+			resp, created = test.CreateChildAreaCreated(t, svc.Context, svc, ctrl, newParentID, ca)
+			// then
+			assert.Equal(t, *ca.Data.Attributes.Name, *created.Data.Attributes.Name)
+			assert.Equal(t, newParentID, *created.Data.Relationships.Parent.Data.ID)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.child2_ok.res.golden.json"), created)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.res.headers.golden.json"), resp.Header())
+		})
+	})
 
-func (rest *TestAreaREST) TestShowAreaOKUsingExpiredIfModifedSinceHeader() {
-	// given
-	_, a := createSpaceAndArea(rest.T(), rest.db)
-	svc, ctrl := rest.SecuredController()
-	// when
-	ifModifiedSince := app.ToHTTPTime(a.UpdatedAt.Add(-1 * time.Hour))
-	res, _ := test.ShowAreaOK(rest.T(), svc.Context, svc, ctrl, a.ID.String(), &ifModifiedSince, nil)
-	//then
-	assertResponseHeaders(rest.T(), res)
+	rest.T().Run("Failure", func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(t, rest.DB, tf.Areas(1))
+		parentArea := fxt.Areas[0]
+		parentID := parentArea.ID
+		childAreaPayload := newCreateChildAreaPayload(uuid.NewV4().String())
+		owner := fxt.Identities[0]
+		svc, ctrl := rest.SecuredControllerWithIdentity(owner)
+		t.Run("Duplicate Child Area", func(t *testing.T) {
+			// when
+			_, created := test.CreateChildAreaCreated(t, svc.Context, svc, ctrl, parentID.String(), childAreaPayload)
+			// then
+			assert.Equal(t, *childAreaPayload.Data.Attributes.Name, *created.Data.Attributes.Name)
+			assert.Equal(t, parentID.String(), *created.Data.Relationships.Parent.Data.ID)
+
+			// try creating the same area again
+			resp, errs := test.CreateChildAreaConflict(t, svc.Context, svc, ctrl, parentID.String(), childAreaPayload)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "conflict.res.golden.json"), errs)
```

Comment:
Can you please make sure that golden files end with `.res.[payload|header].golden.json`?

Reply:
@kwk It already ends with `[res|req].[payload|headers].golden.json`.

Reply:
@jarifibrahim no, it ends with `res.golden.json` and is absent of the `payload` or `headers`.

Reply:
@kwk I didn't know response data is called `response.payload` I thought it's just `response`.
Thank you. I've updated the golden file names in https://github.com/fabric8-services/fabric8-wit/pull/1941/commits/7aceffe07919629a29b5c11bbbb9ace51e744f12